assets/app/components/site/editor/prosemirror.js

```diff
@@ -0,0 +1,55 @@
+
+import React from 'react';
+import prosemirror from 'prosemirror';
+import { exampleSetup, buildMenuItems } from 'prosemirror/dist/example-setup'
+import { defaultMarkdownParser, defaultMarkdownSerializer } from 'prosemirror/dist/markdown';
+import { menuBar } from 'prosemirror/dist/menu';
+import { schema } from 'prosemirror/dist/schema-basic';
+
+const menu = buildMenuItems(schema);
+
+class Prosemirror extends React.Component {
+  constructor(props) {
+    super(props);
+
```

Comment:
I guess yu don't technically have to, but we should toss a
```
this.state = {
  doc: null ( or '')
};
```
in the constructor